dvc/fs/__init__.py

```diff
@@ -87,4 +87,20 @@ def get_cloud_fs(repo, **kwargs):
         remote_conf = SCHEMA["remote"][str](remote_conf)
     except Invalid as exc:
         raise ConfigError(str(exc)) from None
-    return get_fs_cls(remote_conf)(repo, remote_conf)
+
+    if "jobs" not in remote_conf:
+        jobs = repo.config["core"].get("jobs")
+        if jobs:
+            remote_conf["jobs"] = jobs
+
+    if "checksum_jobs" not in remote_conf:
+        checksum_jobs = repo.config["core"].get("checksum_jobs")
+        if checksum_jobs:
+            remote_conf["checksum_jobs"] = checksum_jobs
+
+    cls = get_fs_cls(remote_conf)
+
+    if isinstance(cls, GDriveFileSystem):
+        remote_conf["gdrive_credentials_tmp_dir"] = repo.tmp_dir
+
```

Comment:
These config options would fit nicely into a dynamic config schema, if we had one.